blockchain/txvalidator.go

```diff
@@ -2202,9 +2202,34 @@ func (b *BlockChain) checkCRDPOSManagementTransaction(txn *Transaction) error {
 		return fmt.Errorf("producer already registered")
 	}
 
+	err = b.checkCRDPOSManagementSignature(manager, crMember.DPOSPublicKey)
```

Comment:
The verify public key should be the owner public key of CR not the node public key